frontend/WikiContrib-Frontend/src/components/activity.js

```diff
@@ -7,6 +7,13 @@ import phabricatorPlatformIcon from '../img/phabricatorPlatformIcon.png';
 /**
  * Show all the user Commits on a specific day
  */
+window.history.pushState({page: 1}, "", "");
+
+window.onpopstate = function(event) {
```

Comment:
So, if we include the new history url in the previous statement already, do we need to add this function which is doing no different?

Reply:
I see, rightly said. Including only history.pushState does the job. Thank you for the suggestion :)
I have sent a PR after creating a new branch and making changes there as you mentioned. Hope that solves the issue.

Reply:
Also, now that I have sent the new PR with only the required commit, I can close this PR right?